calico-enterprise/operations/monitor/metrics/recommended-metrics.mdx

```diff
@@ -0,0 +1,514 @@
+| -------------------------------- | ------------------------------------------------------------ |
+| Metric                           | *typha_connections_accepted*                                 |
+| Explanation                      | Total number of connections accepted over time. This value always increases. |
+| Threshold value recommendation   | A steady increase over time is normal. Counters rising after a Felix or Typha restart is also normal (as clients get rebalanced). Investigate connection counters that rise rapidly with no Felix or Typha restarts. |
+| Threshold breach symptoms        | Counters rising when there are no Felix or Typha restarts, or no action that could cause restarts (an upgrade for example), could indicate unexpected Felix or Typha restarts or issues. |
+| Threshold breach recommendations | Check resource usage on Typha(s) and Kubernetes nodes. Increase resources if needed. |
+| Priority level                   | Optional.                                                    |
+
+### Client connections actively streaming
+
+| Client connections actively streaming |                                                              |
+| ------------------------------------- | ------------------------------------------------------------ |
+| Metric                                | *sum by (instance) (typha_connections_streaming)*            |
+| Explanation                           | Current number of active connections that are "streaming" (have completed the handshake), to this Typha. After a connection has been Accepted (reported in the previous metric), there will be a handshake before the connection is deemed to be actively streaming. This indicates how many clients are connected to a Typha. |
+| Threshold value recommendation        | Compare the value for **Total Connections Accepted** and **Client Connections Actively Streaming**. The fluctuation of these values should be in sync with each other if Accepted Connections are turning into Actively Streamed connections. If there is a discrepancy, you should investigate. **Note**: As always, it is recommended to baseline the relationship between these two metrics to have a sense of what is *normal*. |
+| Threshold breach symptoms             | Felix is not getting updates from Typha. {{prodname}} network policies are out of sync. |
+| Threshold breach recommendations      | Check Typha and Felix logs, and rollout restart Typha(s) if needed. |
+| Priority level                        | Recommended.                                                 |
+
+### Rebalanced Client connections
+
+| Rebalanced client connections    |                                                              |
+| -------------------------------- | ------------------------------------------------------------ |
+| Metric                           | *rate(typha_connections_dropped[$__rate_interval])*          |
```

Comment:
I don't think rate_interval is explained?

Reply:
What is it?